deploy/chart/catalog_resources/certified-operators/mongodb-enterprise.v0.3.2.clusterserviceversion.yaml

```diff
@@ -85,8 +79,8 @@ spec:
       group: mongodb.com
       displayName: MongoDB Standalone
       description: MongoDB Deployment consisting of only one host. No replication of data.
-
       resources:
+      # FIXME(alecmerdler): These are probably supposed to be different resources for the different CRD configurations
```

Comment:
@robszumski I feel like these resources should be different depending on if you are creating a `MongoDbStandalone`, `    kind: MongoDbShardedCluster`, or `MongoDbReplicaSet`.